web-local/package.json

```diff
@@ -29,8 +29,8 @@
     "@babel/preset-env": "^7.16.11",
     "@playwright/test": "^1.28.1",
     "@rollup/plugin-typescript": "^8.3.1",
-    "@sveltejs/adapter-static": "^1.0.0-next.41",
-    "@sveltejs/kit": "^1.0.0-next.463",
```

Comment:
Was this a conscious decision? It might have some repercussions.

Reply:
@AdityaHegde would be more aware of these changes.

Reply:
This switches to using stable versions. We can do it in a separate PR.